item/Yonael; Firearms of the Realm.json

```diff
@@ -737,6 +992,1011 @@
 					},
 					"North-West"
 				]
+			],
+			"chapter": {
+				"name": "Missing with Explosives",
+				"ordinal": {
+					"type": "chapter",
+					"identifier": 0
+				},
+				"index": 1
+			}
+		}
+	],
+	"spell": [
+		{
+			"name": "Protection from Normal Missiles",
+			"level": 3,
+			"school": "A",
+			"time": [
+				{
+					"number": 1,
+					"unit": "action"
+				}
+			],
+			"range": {
+				"type": "point",
+				"distance": {
+					"type": "touch"
+				}
+			},
+			"duration": [
+				{
+					"type": "timed",
+					"duration": {
+						"type": "hour",
+						"amount": 1,
+						"upTo": true
+					},
+					"concentration": true
+				}
+			],
+			"classes": {
+				"fromClassList": [
+					{
+						"name": "Artificer",
+						"source": "PHB"
+					},
+					{
+						"name": "Ranger",
+						"source": "PHB"
+					},
+					{
+						"name": "Sorcerer",
+						"source": "PHB"
+					},
+					{
+						"name": "Warlock",
+						"source": "PHB"
+					},
+					{
+						"name": "Wizard",
+						"source": "PHB"
+					}
+				]
+			},
+			"entries": [
+				"Touch a willing creature. They have resistance to non-magical bludgeoning, piercing, and slashing damage from ranged weapons (but not siege weapons)."
+			],
+			"source": "FoTR",
+			"areaTags": [
+				"ST"
+			],
+			"components": {
+				"v": true,
+				"s": true
+			},
+			"page": 6
+		}
+	],
+	"object": [
+		{
+			"name": "Ribald",
+			"source": "",
```

Comment:
should be `"source": "FotR"`

Reply:
Will make the fix.

I changed it to match the shorthand because it was easier to read, and it was closer formatting wise to the homebrew book JSON formatting I used as reference.